src/agents/common.jl

```diff
@@ -90,6 +90,10 @@ mutable struct HEMData
     year_start::ParamScalar
     "Number of hours per representative hour"
     delta_t::ParamScalar
+
+    # inputs for stage 2
+    index_z_local::Dimension
+    index_h_local::Dimension
```

Comment:
I don't think we should need these? I.e., when we're running `LocalDistributionAndDER` `index_z` and `index_h` just are what they are.